shell/platform/tizen/flutter_project_bundle.cc

```diff
@@ -23,9 +25,15 @@ namespace {
 // Returns the path of the directory containing the app binary, or an empty
 // string if the directory cannot be found.
 std::filesystem::path GetExecutableDirectory() {
-  auto res_path = app_get_resource_path();
-  if (res_path == nullptr) {
-    return std::filesystem::path();
+  auto* res_path = app_get_resource_path();
+  if (!res_path) {
+    char buffer[PATH_MAX + 1];
+    auto length = readlink("/proc/self/exe", buffer, sizeof(buffer));
+    if (length > PATH_MAX) {
```

Comment:
readlink can return -1 on error

Reply:
Actually this part is an exact copy of https://github.com/flutter-tizen/engine/blob/flutter-2.10.1-tizen/shell/platform/common/path_utils.cc#L25-L31 and will be replaced with the original function after the x64 support is removed. (It cannot be done right now because of function name ambiguity.)

Reply:
I got you